### DISCUSSION TITLE
SHAP Integration?

### DISCUSSION THREAD

**Discussion by jugro27** [zinggAI/zingg#629] [Q&A]
Is it possible to use SHAP with Zingg? Or is there another preferred/compatible output explanation tool that could be used to describe your model?

**Comment by sonalgoyal** [comment]
We are building the output explanation in the enterprise version, and will add some stuff to the core as well. Do you have any specific needs that you want to discuss?

**Reply by jugro27** [reply]
Awesome, good to know it will be built in eventually! Do you have a timeline on this integration? Also, do you have a direct answer on Zingg's compatibility with SHAP (or any other output explanation tools)?

**Comment by sonalgoyal** [comment]
No, I don’t have a timeline yet. We are planning the next release on the oss in September so  it will be post that. 

I am afraid there is no easy plug into any output explanation tool, we are planning to build something custom for the problem at hand
